app/models/shipit/stack.rb

```diff
@@ -519,7 +519,7 @@ def self.from_param!(param)
     end
 
     delegate :plugins, :task_definitions, :hidden_statuses, :required_statuses, :soft_failing_statuses,
-             :blocking_statuses, :deploy_variables, :filter_task_envs, :filter_deploy_envs,
+             :blocking_statuses, :deploy_variables, :rollback_variables, :filter_task_envs, :filter_deploy_envs,
```

Comment:
Missing new filter delegation.

```suggestion
             :blocking_statuses, :deploy_variables, :rollback_variables, :filter_task_envs, :filter_deploy_envs, :filter_rollback_envs,
```

Reply:
I've added this, but in practice it's not used, since `#build_rollback` is called on a Deploy object directly, not via `Stack`.

It gets even messier. `Deploy` has `delegate :filter_deploy_envs, to: :stack` but it looks like this is also never called, in practice, either :/